src/snowflake/cli/app/__init__.py

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import logging
 import os
 
-# Suppress logging from Snowflake connector
-logging.getLogger("snowflake").setLevel(logging.ERROR)
+from snowflake.cli.app import loggers
+
+loggers.create_initial_loggers()
```

Comment:
Did you test that custom log file path is correctly read (that it is set before invoking this line)?

Reply:
Added test.